src/domains/citizen-auth-app/01_monitor.tf

```diff
@@ -31,3 +40,47 @@ data "azurerm_monitor_action_group" "email" {
   resource_group_name = var.monitor_resource_group_name
   name                = local.monitor_action_group_email_name
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert_v2" "samlresponse_missing_detection_alert" {
+  enabled                 = true
+  name                    = "[${upper(var.domain)}] Missing required SAMLResponse in assertionConsumerService"
+  resource_group_name     = data.azurerm_resource_group.monitor_rg.name
+  scopes                  = [data.azurerm_application_gateway.app_gateway.id]
+  description             = <<-EOT
+    Detected multiple SAMLResponse missing during assertionConsumerService.
+    IdP is unknown here but you can watch Mixpanel events to identify the IdP
+    that is causing the alert to trigger
+  EOT
+  severity                = 0
+  auto_mitigation_enabled = false
```

Comment:
To allow the alert to be triggered once per event, we need to set `auto_mitigation_enabled` to `true`.

Reply:
done in https://github.com/pagopa/io-infra/pull/942/commits/cb9617c2b3285ef5253dc9689216fbdd9a55e71e